src/modules/downloader.js

```diff
@@ -46,58 +48,65 @@ let downloadClientFromRelease = () => {
         fs.mkdirSync('temp', { recursive: true });
         const file = fs.createWriteStream('temp/neutralino.js');
         utils.log('Downloading the Neutralinojs client..');
-        https.get(getClientDownloadUrl(), function (response) {
-            response.pipe(file);
-            file.on('finish', () => {
-                file.close();
-                resolve();
-            });
-        });
+
+        fetch(getClientDownloadUrl()).then(res => {
```

Comment:
I would like to vote for the built-in `http` since, at this level, we just need to download a file from internet. Also, the `http` package works as expected and solves the current requirement :) Perhaps, if we need to call REST APIs etc someday, we can think about an abstract lib like `node-fetch`